crates/pallet-executor/src/lib.rs

```diff
@@ -194,16 +216,34 @@ mod pallet {
                         .propagate(true)
                         .build()
                 }
+                Call::submit_fraud_proof { fraud_proof } => {
+                    // TODO: proper error value
+                    const INVALID_DRAUD_PROOF: u8 = 100;
```

Comment:
I prefer having such constants at the top of the file. Also you could have used an enum with explicit discriminants here.